Services/housekeeping_service.c

```diff
@@ -87,10 +90,10 @@ SAT_returnState ground_response_task(csp_packet_t *packet){
     portBASE_TYPE err;
     TC_TM_app_id server_address = GND_APP_ID;
     /*To get conn from the response queue*/
-    if((err = xQueueReceive(response_queue, conn, 
+    if((err = xQueueReceive(response_queue, conn,
```

Comment:
@iBacklight This is a good learning moment:

This won't quite work properly because a queue in FreeRTOS will wake up the host task from the sleeping state when data is put into the queue. This is a useful way to both send data between tasks, and synchonize data access. In this case it won't quite work because the function `ground_response_task` isn't quite a function, for 3 reasons:
1. it's return type isn't void
2. it's parameter isn't (void*)
3. it returns - a task should have an infinite loop
Also, it's never started as a task. I'll show you what I mean in a bit!